service/route53/handwritten_paginators_test.go

```diff
@@ -0,0 +1,113 @@
+package route53
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/route53/types"
+	"github.com/aws/smithy-go/middleware"
+	"testing"
+)
+
+var limit int32
+var startRecordName *string
+var startRecordIdentifier *string
+var startRecordType types.RRType
+
+type testListRRSMiddleware struct {
+	id int
+}
+
+func (m *testListRRSMiddleware) ID() string {
+	return fmt.Sprintf("mock middleware %d", m.id)
+}
+
+func (m *testListRRSMiddleware) HandleInitialize(ctx context.Context, input middleware.InitializeInput, next middleware.InitializeHandler) (
```

Comment:
**fix**: Same comment as s3 paginators, this can be done with a mock client more cleanly